modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/lifecycle/MultipleReleaseInstanceResourceProvider.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.vulcan.internal.jaxrs.lifecycle;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.cxf.jaxrs.lifecycle.ResourceProvider;
+import org.apache.cxf.message.Message;
+
+/**
+ * @author Alejandro Tardín
+ */
+public class MultipleReleaseInstanceResourceProvider
```

Comment:
I am not sure about this naming :thinking: 

What do you think about `SafeReleaseInstanceResourceProvider` or `SafeInstanceReleaserResourceProvider`?

Reply:
See e0d8e0bf6589a5e0058062659d42900f980f3912.